.github/workflows/release.yaml

```
@@ -66,7 +66,7 @@ jobs:
           builder: ${{ steps.buildx.outputs.name }}
           context: .
           file: ./Dockerfile
-          platforms: linux/amd64 #,linux/arm/v7,linux/arm64
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
```

Please remove arm/v7 for the moment.
Production clusters would use only amd64 and arm64.

```suggestion
          platforms: linux/amd64,linux/arm64 # linux/arm/v7
```